### PR TITLE
Rodrigo menacho/camelcasemethod

### PR DESCRIPTION
**Pull request by EotronRM**

### CODE REVIEW

File: config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -194,4 +182,4 @@
 
     </module>
 
-</module>
```
Comment (carledriss) [inline config/checkstyle/checkstyle.xml]: Why did you modify this config file ?

File: src/main/java/org/fundacionjala/coding/camelcase/Solution.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+     * @param str **Input String**
+     * @return **return the String in CamelCase**
+     */
+    public static String camelCase(final String str) {
+
+        String[] words = str.trim().split(" ");
+        for (int i = 0; i < words.length; i++) {
+            if (!words[i].isEmpty()) {
+                if (words[i].length() != 0) {
+                    words[i] = words[i].substring(0, 1).toUpperCase() + words[i].substring(1);
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/camelcase/Solution.java]: Don't concatenate strings using +

File: config/checkstyle/checkstyle.xml (hunk 1 above)
Comment (Warchino) [reply config/checkstyle/checkstyle.xml]: I didn't modify that archive.

**Review by carledriss — CHANGES_REQUESTED** [review]
Don't forget to increase the code coverage

**Review by carledriss — CHANGES_REQUESTED** [review]
Please increase the code coverage.
https://codecov.io/gh/AT-08/coding/pull/35/diff?src=pr&el=tree#diff-c3JjL21haW4vamF2YS9vcmcvZnVuZGFjaW9uamFsYS9jb2RpbmcvY2FtZWxjYXNlL1NvbHV0aW9uLmphdmE=

**Review by carledriss — APPROVED** [review]
LGTM! 👍

**Pull request merged**